client-js/app/scripts/app.ts

```diff
@@ -141,13 +141,20 @@ window.addEventListener('WebComponentsReady', () => {
 
   class RegistryActivity implements Activity {
     path: Array<string>;
+    private globElem: LabradRegistry;
```

Comment:
I don't see how changing the name could lead to that error. Seems like you must have been doing something different in the `start` method to get that. Maybe it just wasn't a clean compile?